docs/ref/checks.txt

```diff
@@ -448,6 +448,8 @@ The following checks are performed on your URL configuration:
   able to reverse all URLs in this namespace.
 * **urls.E006**: The :setting:`MEDIA_URL`/ :setting:`STATIC_URL` setting must
   end with a slash.
+* **urls.E007**: The custom ``handlerXXX`` view must take the correct number of
+  arguments (...).
```

Comment:
There was a ticket in review for awhile to replace `...` with the ellipses character. Anyone know if that has gone through yet or not?

Reply:
There are `...` entries in `docs/releases/2.2.txt` so I'm guessing not.

Reply:
It was merged, but it wasn't a repo wide scan/replace. 

* We should probably favour `…`.
* We should probably document favouring `…` 
* We're still coders. `...` has to be allowed no? 😀

Reply:
I've done it…